core/server/worker/src/test/java/alluxio/worker/block/PinListSyncTest.java

```diff
@@ -29,26 +28,33 @@
 
 public class PinListSyncTest {
 
+  // context for creating FileSystemMasterClient
+  private final MasterClientContext mContext = MasterClientContext.newBuilder(ClientContext.create(
+      Configuration.global())).build();
+
   private PinListSync mSync;
   private TestBlockWorker mBlockWorker;
   private FileSystemMasterClient mFileSystemMasterClient;
-  private final Closer mCloser = Closer.create();
 
   @Before
   public void before() {
     mBlockWorker = new TestBlockWorker();
```

Comment:
Just initialize this at the place where this is declared.

Reply:
Fixed